### PR TITLE
[SIE-130] add SEO tags and favicon

### PR DESCRIPTION
**Pull request by alecherryy**
Updated logos, favicon and meta tags for better SEO and more descriptive content. Jira [ticket](https://cs6510.atlassian.net/browse/SIE-130).

### CODE REVIEW

**Review by quanganhhoang — APPROVED** [review]
AWESOME!!

File: public/manifest.json [hunk 1]
```diff
@@ -10,16 +10,16 @@
     {
       "src": "logo192.png",
```
Comment (fernandowinfield) [inline public/manifest.json:11]: This is the logo I am talking about

**Review by fernandowinfield — APPROVED** [review]
This all looks awesome!

I just have a curiosity about the logo 192 I where I left a comment; will it matter that the file you added is .svg and the manifest says .png?

File: public/manifest.json (hunk 1 above)
Comment (alecherryy) [reply public/manifest.json:11]: AH! Good catch, I didn't notice that. Thanks

**Comment by alecherryy** [comment]
> This all looks awesome!
> 
> I just have a curiosity about the logo 192 I where I left a comment; will it matter that the file you added is .svg and the manifest says .png?

Fixed.

**Pull request merged**